docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '3'
+
+networks:
+    net:
+        name: net
+
+volumes:
+    aica:
+
+services:
+    db:
+        image: postgresql
```

Comment:
Specify the version (14) so the docker-compose suddenly break in the future due to too new version